assets/js/blocks/reviews/all-reviews/index.js

```diff
@@ -51,6 +51,22 @@ registerBlockType( 'woocommerce/all-reviews', {
 		},
 	},
 
+	transforms: {
+		from: [
+			{
+				type: 'block',
+				blocks: [ 'core/legacy-widget' ],
+				// We can't transform if raw instance isn't shown in the REST API.
+				isMatch: ( { idBase, instance } ) =>
+					idBase === 'woocommerce_recent_reviews' && !! instance?.raw,
+				transform: ( { instance } ) =>
+					createBlock( 'woocommerce/all-reviews', {
+						reviewsOnPageLoad: instance.raw.number,
```

Comment:
Our Reviews block seems to work a bit differently from the widget (ie: it shows the reviewer picture instead of the product picture by default). What do you think about adding some more attributes here to make the block feel closer to the widget? These are the ones I thought might help:
```suggestion
						reviewsOnPageLoad: instance.raw.number,
						imageType: 'product',
						showLoadMore: false,
						showOrderby: false,
						showReviewDate: false,
						showReviewContent: false,
```

Reply:
Great suggestion, definitely improves the experience! :)